### DISCUSSION TITLE
Show gradient in percent?

### DISCUSSION THREAD

**Discussion by OleKristianH** [SauceLLC/sauce4zwift#35] [Q&A] [answered]
I can't find this anywhere. Is it supported?

**Comment by mayfield** [comment, marked as answer]
Not supported currently.  I'm not doing any map or topo related functions yet.

**Reply by OleKristianH** [reply]
Ok. Thanks. This is the only thing I'm missing when I have the HUD turned off. I guess the gradient is written to log.txt as:
`[19:13:42] [FTMS] SIM Grade -0.16% --> -0.27%`
